src/query/query.cpp

```diff
@@ -108,15 +108,24 @@ std::vector<properties::Property*> allProperties() {
         p.end()
     );
 
+    std::vector<properties::Property*> q =
+        OsEng.virtualPropertyOwner().propertiesRecursive();
+
+    properties.insert(
```

Comment:
Since `allProperties` are used from`property_setValueRegex`, it seems a bit strange that virtual properties (which are already regex-formatted) are also included in the search. Is there a reason why virtual properties are added to this list?